tests/lax_scipy_sparse_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from functools import partial
+from absl.testing import parameterized
+
+import numpy as onp
+import scipy.sparse.linalg as osp_sparse
+from jax import lax
+from jax import test_util as jtu
+from jax.experimental import sparse
+
+@parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}D_{}".format(dim, onp.dtype(dtype).name),
+       "dim": dim, "dtype": dtype}
+      for dim in [3, 5, 10]
+      for dtype in [onp.float32, onp.float64]))
+def test_linalg_sparse_cg(self, dim, dtype):
```

Comment:
This needs to be in a class in order for it to be tested. Right now I suspect your tests are not running :)